src/app/content/practiceQuestions/components/ShowPracticeQuestions.spec.tsx

```diff
@@ -151,4 +154,25 @@ describe('ShowPracticeQuestions', () => {
 
     expect(() => component.root.findByType(Question)).not.toThrow();
   });
+
+  it('renders FinalScreen screen', () => {
+    store.dispatch(receiveBook(book));
+    store.dispatch(setSelectedSection(linkedArchiveTreeSection));
+    store.dispatch(receivePracticeQuestionsSummary({
+      countsPerSource: { [linkedArchiveTreeSection.id]: 3 },
+    }));
+
+    store.dispatch(setQuestions([{ id: 'asd' } as any as PracticeQuestion]));
+    store.dispatch(setAnswer({ questionId: 'asd', answer: { id: 'qwe' } as any as PracticeAnswer }));
+
+    store.dispatch(nextQuestion());
+    const component = renderer.create(render());
```

Comment:
please group `store.dispatch` calls together 
I also think that you could `store.dispatch(finishQuestions())` instead of dispatching `nextQuetions`